guacamole-ext/src/main/java/org/apache/guacamole/net/auth/simple/SimpleUserContext.java

```diff
@@ -99,7 +129,7 @@ public SimpleUserContext(AuthenticationProvider authProvider,
             GuacamoleConfiguration config = configEntry.getValue();
 
             // Add as simple connection
-            Connection connection = new SimpleConnection(identifier, identifier, config);
+            Connection connection = new SimpleConnection(identifier, identifier, config, true);
```

Comment:
Should this be `true`, or should this be passing the value of `interpretTokens`?

Reply:
Good catch - we should pass through `interpretTokens`.